pkgs/by-name/an/ant/package.nix

```diff
@@ -11,6 +11,8 @@ stdenv.mkDerivation (finalAttrs: {
   pname = "ant";
   version = "1.10.15";
 
+  buildInputs = [ jre ];
```

Comment:
> Also, will this change make it so that we don't actually need to add both jdk and ant into nativeBuildInputs when using ant?
> 
> Also also, `jdk` vs `jre`? (I'd say `jdk`)
> --@TomaSajt

> From my testing, if we use `jdk`, that is the case, but setting it to `jdk` makes it difficult to use a different version of Java without overriding ant.
> --Me

> Is there a reason that we shouldn't have consumers explicitly specify a JDK?
> --Me